dev/com.ibm.ws.channelfw/src/com/ibm/ws/bytebuffer/internal/WsByteBufferPoolManagerImpl.java

```diff
@@ -92,7 +92,7 @@
     private long lastTimeCheck = 0L;
     private int leakDetectionInterval = -1;
     private String leakDetectionOutput = null;
-    private final Integer leakDetectionSyncObject = new Integer(1);
+    private final Object leakDetectionSyncObject = new Object() {};
```

Comment:
Why switching to `new Object()` here? AFAIK the `new Object()` constructor might be removed in a future Java release, so we should avoid new usages of this.

Reply:
It is making an anonymous class that extends Object.  If that won't work, I can make it a static inner class like in the other classes.  I may have some additional classes to change if we want to remove creating anonymous inner classes that way.